algobot/__main__.py

```diff
@@ -16,6 +16,7 @@
                              QMainWindow, QMessageBox, QTableWidgetItem)
 
 import algobot.assets
+from algobot import helpers
```

Comment:
Why import helpers when we can import PATHS from algobot.helpers?

Reply:
Would you rather all usages of PATHS to be called directly?

Reply:
I guess it'd make the most sense to have PATHS in another file and then import <other_file>.Paths and use it

Reply:
Do we still need this import by the way?

Reply:
Updated